README.md

```diff
@@ -63,21 +63,21 @@
 - Open PowerShell with Administrator privileges. Run `Set-ExecutionPolicy RemoteSigned`. Type Y and press Enter to confirm that you want to make this change. (If you don't perform this step, you will not be able to run **any** PowerShell scripts on your system.)
 - Install the [Chocolatey package manager for Windows](https://chocolatey.org/install).
 - Reopen PowerShell with Administrator privileges, if necessary. Run the [install-apps-via-chocolatey.ps1](/best-practices/development-tools/windows-setup/install-apps-via-chocolatey.ps1) script to install a number of programs you will probably need. (Feel free to peek at this file's contents to see what it contains.)
-- (Optional) From the same PowerShell window, run the command `oosu10`. Here, you can easily configure a number of settings to have Windows respect your privacy a little more. Just make sure you understand what each setting does before you change it.
+- (Optional) From the same PowerShell window, run the command `oosu10`. Here, you can easily configure several settings to have Windows respect your privacy a little more. Just make sure you understand what each setting does before you change it.
 - After running `oosu10`, restart Windows (if applicable).
-- Install Android Studio.
+- Install Android Studio
 - Install the regular version of Visual Studio (as opposed to Visual Studio Code). Visual Studio Community Edition may be sufficient, or you may need a paid license for Visual Studio Professional, Enterprise, or similar. Either way, ask your PM (Project Manager).
 - You may wish to use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/faq).
 - Ask your Project Manager if you need to install anything else.
 
 Using Chocolatey has several advantages:
 1. You can install most of the software you need just by running a single script.
-2. You can update all of this software later, by running `choco upgrade all`, as Administrator.
+2. You can update all of this software later; by running `choco upgrade all`; as Administrator.
```

Comment:
I feel like the commas shouldn't be there in the first place and the semicolons  don't belong here either. The sentence isn't long enough to warrant a pause, they are not two independent clause, and they are not separated by a conjunction.

Reply:
I agree with @B12Howard, I think we can just remove both semicolons and commas here.